### ISSUE TITLE
Accessibility issues with charts ticks and legends 

### ISSUE THREAD

**Issue by reeteshsingh93** [carbon-design-system/carbon-charts#536]
#### I am submitting a...

Accessibility defect

#### charts version:

"@carbon/charts": "^0.29.6",
"@carbon/charts-react": "^0.29.6",

### Issue description

Elements having certain WAI-ARIA navigation landmark role or the document structure role must have a label specified with aria-label or aria-labelledby
The g element with a WAI-ARIA grouping role group does not have the required label specified by 'aria-label' or 'aria-labelledby'.

DAP rule set: IBM Accessibility January 2020

#### Steps to produce the issue

Go to https://dev.console.test.cloud.ibm.com/security-advisor#/configadvisor

Screenshots:

![image](https://user-images.githubusercontent.com/46310374/76708060-03df5500-671a-11ea-8068-ea0480db95c2.png)
![image](https://user-images.githubusercontent.com/46310374/76708063-08a40900-671a-11ea-86eb-9197bf4eaf63.png)

#### Current behavior

Accessibility issues being reported by DAP

#### Expected behavior
No voilation should be reported by DAP plugin

**Comment by reeteshsingh93** [comment]
Can we have any update on this?

**Comment by sophiiae** [comment]
Closed by #707
